src/ExpressionEvaluator/Core/Source/ExpressionCompiler/CustomTypeInfo.cs

```diff
@@ -1,37 +1,196 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using Microsoft.CodeAnalysis.Collections;
+using Microsoft.VisualStudio.Debugger.Evaluation.ClrCompilation;
 using System;
 using System.Collections.ObjectModel;
-using Microsoft.VisualStudio.Debugger.Evaluation.ClrCompilation;
+using System.Text;
 
 namespace Microsoft.CodeAnalysis.ExpressionEvaluator
 {
-    /// <remarks>
-    /// We can't instantiate <see cref="DkmClrCustomTypeInfo"/> in the unit tests
-    /// (since we run against a reference assembly), so we use this type as an
-    /// intermediary.
-    /// </remarks>
-    internal struct CustomTypeInfo
+    internal static class CustomTypeInfo
```

Comment:
`DkmClrCustomTypeInfo` is a type defined externally, and this type was previously a `struct` with a similar shape to `DkmClrCustomTypeInfo` so the comment was meaningful. Now the type is a simply a collection of `static` helper methods.